files/default/sql/ddl/updates/1.4.0.sql

```diff
@@ -44,4 +44,28 @@ DROP TABLE IF EXISTS `hopsworks`.`tf_lib_mapping`;
 ALTER TABLE `hopsworks`.`project` ADD COLUMN `docker_image` varchar(255) COLLATE latin1_general_cs DEFAULT NULL;
 ALTER TABLE `hopsworks`.`project` DROP COLUMN `conda_env`;
 
-ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN `docker_image`;
\ No newline at end of file
+ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN `docker_image`;
+
+ALTER TABLE `conda_commands` DROP FOREIGN KEY `user_fk`, ADD CONSTRAINT `user_fk_conda` FOREIGN KEY (`user_id`) REFERENCES `users` (`uid`) ON DELETE CASCADE ON UPDATE NO ACTION;
```

Comment:
For consistency can you add `hopsworks` before the table names?

Reply:
fxd